docs/admin/getting-started/other/bare-metal.md

```diff
@@ -7,12 +7,12 @@ description: 'Manual, minimalist setup with essential features.'
 
 ## User Guide for Installing OpenCloud
 
-Follow the steps below to install and configure OpenCloud on your system.<br/>
-This example is on Linux Ubuntu 24.04 distribution!
+Follow the steps below to install and configure OpenCloud on your system.<br/> This example is on Linux Ubuntu 24.04
```

Comment:
On a general note: Do we need to use `<br/>` in markdown files? Usually an empty lines results in such.

```suggestion
Follow the steps below to install and configure OpenCloud on your system.

This example is on Linux Ubuntu 24.04
```